trax/data/tf_inputs.py

```diff
@@ -1090,30 +1095,136 @@ def download_and_prepare(dataset_name, data_dir):
+      segment_embs = np.zeros(sent1.shape[0] + 2, dtype=np.int32)
+      yield value_vector, segment_embs, segment_embs, label, np.int64(1)
+  else:
+    for sent1, *_ in batch: # row is a tuple with 1 element
+      value_vector = np.concatenate(([CLS_id], sent1, [SEP_id]))
+      segment_embs = np.zeros(sent1.shape[0] + 2, dtype=np.int32)
+      yield value_vector, segment_embs, segment_embs
 
 
-def bert_double_sentence_preprocess(batch):
-  for sent1, sent2, label in batch:
-    value_vector = np.concatenate(([101], sent1, [102], sent2, [102]))
+def BertDoubleSentenceInputs(batch, labeled=True, CLS_id=101, SEP_id=102):
+  """Prepares inputs for BERT models by adding [SEP] and [CLS] tokens and creating segment embeddings."""
+  if labeled:
+    for sent1, sent2, label in batch:
+      value_vector = np.concatenate(([CLS_id], sent1, [SEP_id], sent2, [SEP_id]))
+
+      segment_embs = np.zeros(sent1.shape[0] + sent2.shape[0] + 3, dtype=np.int32)
+      second_sent_start = sent1.shape[0] + 2
+      segment_embs[second_sent_start:] = 1
+      yield value_vector, segment_embs, segment_embs, label, np.int64(1)
+  else:
+    for sent1, sent2 in batch:
+      value_vector = np.concatenate(([CLS_id], sent1, [SEP_id], sent2, [SEP_id]))
 
-    segment_embs = np.zeros(sent1.shape[0] + sent2.shape[0] + 3, dtype=np.int32)
-    second_sent_start = sent1.shape[0] + 2
-    segment_embs[second_sent_start:] = 1
-    yield value_vector, segment_embs, segment_embs, label, np.int64(1)
+      segment_embs = np.zeros(sent1.shape[0] + sent2.shape[0] + 3, dtype=np.int32)
+      second_sent_start = sent1.shape[0] + 2
+      segment_embs[second_sent_start:] = 1
+      yield value_vector, segment_embs, segment_embs
 
 
-def CreateBertInputs(double_sentence=True):  # pylint: disable=invalid-name
-  """Prepares inputs for BERT models by adding [SEP] and [CLS] tokens and creating segment embeddings."""
-  if double_sentence:
-    return bert_double_sentence_preprocess
-  else:
-    return bert_single_sentence_preprocess
+def CreateBertInputs(double_sentence=True, labeled=True, CLS_id=101, SEP_id=101):
+  foo = BertDoubleSentenceInputs if double_sentence else BertSingleSentenceInputs
+  return functools.partial(foo, labeled=labeled, CLS_id=CLS_id, SEP_id=SEP_id)
+
 
+def mask_random_tokens(batch, vocab_size=30522, masking_prob=0.15,
+                       CLS_id=101, SEP_id=102, MASK_id=103, vocab_start_id=999
+                       ):
+  """
+  Prepares input for masking task by masking masking_prob % of non-special tokens at each input row.
+    round(masking_prob * num_nonspecial_tokens) random tokens are selected out of which each token is either
+      - replaced with [MASK] token with 80% probability
+      - replaced with random token with 10% probability
+      - unchanged with 10%
+  Follows standard procedure of masking in BERT found in
+  create_masked_lm_predictions-https://github.com/google-research/bert/blob/master/create_pretraining_data.py, line 342.
```

Comment:
We haven't switched on the linter externally due to some issues, but can you keep the comments (and code lines) to 80 characters?

Thanks!

Reply:
Oh, sorry! I have a vertical line in my IDE that shows me the limit of 80 chars. It looked it's still less, I guess even coming that close to 80 chars was unnecessary.